src/ECS/System/BasicSystem.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <entt/entt.hpp>
+
+#include "System/Logger.h"
+
+namespace Sourcehold {
+    namespace ECS {
+        namespace System {
+            template<typename ...Selector>
```

Comment:
Selector are components?

Reply:
Yes :)
I can rename that, but semantically and logically... I feel that it is better to leave it as it is.

Reply:
Yes :)
I can rename that, but semantically and logically... I feel that it is better to leave it as it is.